src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -146,7 +145,7 @@ public function execute(WorkflowInstance $workflow)
                     $item->Assignee = ArrayData::create($assigneeVars);
                 }
 
-                $body = $view->process($item);
+                $body = $engine->renderString($this->EmailTemplate, ViewLayerData::create($item));
```

Comment:
The `EmailTemplate` value isn't hardcoded, so could be updated at a project level to be whatever template syntax the devs want to use, therefore use `TemplateEngine` not explicitly `SSTemplateEngine`